### PR TITLE
feature: adds autoexit option. Default: do autoexit after script

### PR DESCRIPTION
**Pull request by eljefedelrodeodeljefe**
@davidmarkclements maybe this would be interesting for you. Don't know what your plan for this was, but I like the notion of autoexiting, after the script has run. It will however wind down the rest of the app logic.

### CODE REVIEW

File: examples/simple/index.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
 'use strict'
-let count = 0, max = 10, res = 0
+let count = 0
+let max = 10
+let res = 0
```
Comment (davidmarkclements) [inline examples/simple/index.js]: let -> var

**Comment by davidmarkclements** [comment]
Yah this is great, I don't think we need the option, just do it on exit, otherwise why use 0x in the firstlace

File: examples/simple/index.js (hunk 1 above)
Comment (eljefedelrodeodeljefe) [inline examples/simple/index.js]: will do.

**Comment by eljefedelrodeodeljefe** [comment]
Then I'll remove the option.

**Comment by eljefedelrodeodeljefe** [comment]
Done.

**Comment by davidmarkclements** [comment]
Outstanding thanks man

**Comment by davidmarkclements** [comment]
just need to check/get linux behaviour the same on this

**Comment by eljefedelrodeodeljefe** [comment]
Sure. I have parallels installed. Will check too.

**Comment by eljefedelrodeodeljefe** [comment]
Apologies if I missed it.

**Comment by eljefedelrodeodeljefe** [comment]
I know where the issues lie. See comments. basically I was mistaken the start function to be global. whereas it should be fine send `process.kill(proccess.pid, 'SIGINT')` in the exit event of the spawn functions directly. I'll adda global function and call at in `sun()` and `linux()` respectively.

**Comment by davidmarkclements** [comment]
yah cool - by global you mean scoped of course - not actually adding to `global`

**Comment by eljefedelrodeodeljefe** [comment]
Call it [here](https://github.com/davidmarkclements/0x/pull/15/files#diff-168726dbe96b3ce427e7fedce31bb0bcR66) and [here](https://github.com/davidmarkclements/0x/pull/15/files#diff-168726dbe96b3ce427e7fedce31bb0bcR189)

**Comment by eljefedelrodeodeljefe** [comment]
Yes. Scope in the file. Actually extra function won't be needed. I'll send in the thing later.

**Comment by eljefedelrodeodeljefe** [comment]
This shows it's behaviour https://www.youtube.com/edit?o=U&video_id=QzQa34qZKOU

**Comment by davidmarkclements** [comment]
link is broken (maybe just visible to you if logged in)

**Comment by eljefedelrodeodeljefe** [comment]
This one, sorry https://youtu.be/QzQa34qZKOU

**Pull request merged**